ultralytics/yolo/engine/model.py

```diff
@@ -156,6 +156,7 @@ def val(self, data=None, **kwargs):
         overrides = self.overrides.copy()
         overrides.update(kwargs)
         overrides["mode"] = "val"
+        overrides["val_split"] = overrides.get("val_split", "val") or "val"
```

Comment:
Hey, @validatedev can you confirm if this line is needed? I think the default value will already be val so this might be redundant.

Reply:
Yes you are completely right. Fixing.